### PR TITLE
Ruby 2.7.0-rc1 Released

### PR DESCRIPTION
**Pull request by nurse**

### CODE REVIEW

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
+---
+layout: news_post
+title: "Ruby 2.7.0-rc1 Released"
+author: "naruse"
+translator:
+date: 2019-12-17 12:00:00 +0000
+lang: en
+breaks: false
```
Comment (stomar) [inline en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: What's the idea behind this tag?

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+It also provides rdoc integration. In `irb` you can display the reference
+for a given class, module, or method.
+[[Feature #14683]](https://bugs.ruby-lang.org/issues/14683),
+[[Feature #14787]](https://bugs.ruby-lang.org/issues/14787),
+[[Feature #14918]](https://bugs.ruby-lang.org/issues/14918)
+
+Besides, source lines shown at `binding.irb` and inspect results
+for core-class objects are now colorized.
+
+<video autoplay="autoplay" controls="controls" muted="muted" width="576" height="259">
```
Comment (stomar) [inline en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: autoplay?

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+* Require compilers to support C99
+  [[Misc #15347]](https://bugs.ruby-lang.org/issues/15347)
+  * Details of our dialect:
+    <https://bugs.ruby-lang.org/projects/ruby-trunk/wiki/C99>
+
+* ~~`Regexp#match{?}` with `nil` raises TypeError as String, Symbol.
+  [[Feature #13083]](https://bugs.ruby-lang.org/issues/13083)~~ reverted
+
+3895 files changed, 213426 insertions(+), 96934 deletions(-)
```
Comment (stomar) [inline en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: duplicate of stats further below

File: _data/releases.yml [hunk 4]
```diff
@@ -21,6 +21,38 @@
 
 # 2.7 series
 
+- version: 2.7.0-rc1
+  date: 2019-12-18
+  post: /en/news/2019/12/18/ruby-2-7-0-rc1-released/
+  total_files: 4163
```
Comment (stomar) [inline _data/releases.yml]: possibly more clearer:

```yaml
stats:
  files_changed: 4163
  insertions: 226280
  deletions: 99449
```

(I could adjust that at a later time.)

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+date: 2019-12-17 12:00:00 +0000
+lang: en
+breaks: false
+---
+
+{% assign release = site.data.releases | where: "version", "2.7.0-rc1" | first %}
+
+We are pleased to announce the release of Ruby 2.7.0-rc1.
+
+A preview version is released to gather feedback for the final release
```
Comment (stomar) [inline en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: preview version?

**Comment by stomar** [comment]
@nurse

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md (hunk 1 above)
Comment (nurse) [reply en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: It's for HackMD which is used to edit draft but the tag will be removed in next version.

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md (hunk 5 above)
Comment (nurse) [reply en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: it should be fixed as "A release candidate is ..."

File: _data/releases.yml (hunk 4 above)
Comment (nurse) [reply _data/releases.yml]: I fix it in next release

Comment (stomar) [reply _data/releases.yml]: Already fixed for this post.

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md (hunk 3 above)
Comment (stomar) [reply en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: fixed

File: en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md (hunk 5 above)
Comment (stomar) [reply en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md]: fixed

**Pull request merged**